tests/spirv/push-constant-layout.slang

```diff
@@ -0,0 +1,26 @@
+//TEST:SIMPLE(filecheck=CHECK): -target spirv -fvk-use-entrypoint-name
+// CHECK-NOT: std140
+struct Transform
+{
+    float4 Tint;
+    float2x2 ScaleRot; // << why u no std430???
```

Comment:
Is that comment from before the bug was fixed perhaps?

Reply:
Hah. That got copied in from my bug report. :grin: